magefile.go

```diff
@@ -283,6 +298,22 @@ func IntegrationFull() error {
 		"test/integration")
 }
 
+// Run full set of integration tests.
+func IntegrationFullNoDocker() error {
+	fmt.Println("Running all integration tests...")
+
+	return sh.RunV(pythonExecutableName, "-m", "pytest", "-m", "not slow_ee and not notarantool and not docker",
+		"test/integration")
+}
+
+// Run full set of integration tests.
```

Comment:
This does not look like full integration set, but only docker.

Reply:
changed